File: XUnit/SauceryX/Capabilities/ConcreteCreators/DesktopCreator.cs
using SauceryX.Capabilities.Base;
using SauceryX.Capabilities.ConcreteProducts;
using SauceryX.OnDemand;

namespace SauceryX.Capabilities.ConcreteCreators {
    internal class DesktopCreator : Creator {
        public override BaseCapabilities Create(SaucePlatform platform, string testName) {
            return new DesktopCapabilities(platform, testName);
        }
    }
}
/*
 * Copyright Andrew Gray, Full Circle Solutions
 * Date: 18th September 2014
 * 
 */

File: JUnit/saucery-core/src/main/java/au/com/fullcirclesolutions/saucery/capabilities/DesktopCreator.java
package au.com.fullcirclesolutions.saucery.capabilities;

class DesktopCreator extends Creator {
    @Override
    public BaseCapabilities Create(SaucePlatform platform, String testName) {
        return new DesktopCapabilities(platform, testName);
    }
}
/*
 * Copyright: Andrew Gray, Full Circle Solutions
 * Date: 18th September 2014 
 */